content/backmatter.tex

```diff
@@ -157,6 +157,12 @@ \chapter{Undefined Behavior in OpenSHMEM}\label{sec:undefined}
 an error message saying that the \ac{PE} accessed is outside the range of
 accessible \acp{PE}, or may exit without a warning.\tabularnewline
 \hline
+Accessing an invalid team member \ac{PE} & Passing a \ac{PE} value that
+corresponds to an invalid team member to an RMA/AMO routine, as the root of
+a broadcast, etc. results in undefined behavior.  Invalid team member
+\ac{PE} values include those that are negative or greater than or equal to the size of
+the team.\tabularnewline
+\hline
```

Comment:
I think this case should be folded in with the previous one. Your wording is more precise, but _any_ OpenSHMEM routine that takes a PE number refers to some PE in some team. It just happens that our "legacy" APIs use the world team.

I tried to do a multiline suggestion for both, but GitHub won't let me. (Although it now supports multi-line comments!)

Reply:
I'll suggest the following (for both cases, combined):

> **Specifying invalid PE numbers**
> For OpenSHMEM routines that accept a PE number as an argument, if the PE number is invalid for the team associated with the operation (either implicitly or explicitly), the behavior is undefined. An invalid PE number include those that are negative or greater than or equal to the size of the associated team.

Reply:
Yeah, I like it.  We'd lose the "library may handle this situation [however it wants]...", but I presume this is understood when we declare that something has undefined behavior, right?

One minor typo is "include" -> "includes".  I'll push this change.